SecurityPkg/Test/SecurityPkgHostTest.dsc

```diff
@@ -28,7 +28,6 @@
   SecurityPkg/Test/Mock/Library/GoogleTest/MockPlatformPKProtectionLib/MockPlatformPKProtectionLib.inf
   SecurityPkg/Library/DxeTpm2MeasureBootLib/InternalUnitTest/DxeTpm2MeasureBootLibSanitizationTestHost.inf
   SecurityPkg/Library/DxeTpmMeasureBootLib/InternalUnitTest/DxeTpmMeasureBootLibSanitizationTestHost.inf
-
   #

```

Comment:
No change is required in DSC.

Reply:
The code change in SecurityPkgHostTest.dsc is removed in the new commit. Thanks for the suggestion.